packages/component/src/Utils/AbsoluteTime.js

```diff
@@ -6,14 +6,20 @@ import ScreenReaderText from '../ScreenReaderText';
 import useLocalize from '../hooks/useLocalize';
 import useLocalizeDate from '../hooks/useLocalizeDate';
 
-const AbsoluteTime = ({ value }) => {
+const CalculateAbsoluteTime = value => {
   const localizedTime = useLocalizeDate(value);
-  const text = useLocalize('SentAt') + localizedTime;
+
+  return localizedTime;
+};
+
+const AbsoluteTime = ({ value }) => {
+  const absTime = CalculateAbsoluteTime(value);
```

Comment:
```suggestion
  const absTime = useLocalizeDate(value);
```

Reply:
These are different, lol.

Reply:
How are they different? William recommended the same thing.